<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditAuthorPage</class>
 <widget class="QMainWindow" name="EditAuthorPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="nameCombo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>性别:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="genderCombo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>男</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>女</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>其他</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>单位:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="unitEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>研究领域:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="fieldCombo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>农林牧渔、卫生、科学研究</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>建筑、能源、冶炼、交通运输</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>制造、信息技术、贸易</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>党政、社团、国防、法律、金融</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>教育、公共文化、社会服务</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QListWidget" name="documentsListView">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>261</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>16</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>拥有或参与的文献</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>移除</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteAuthor">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>331</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除该作者</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>545</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>EditAuthorPage</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
